src/sidebar/directive/annotation-share-dialog.js

```diff
@@ -4,73 +4,69 @@ var angular = require('angular');
 
 var scopeTimeout = require('../util/scope-timeout');
 
-module.exports = function () {
-  return {
-    bindToController: true,
-    controllerAs: 'vm',
-    restrict: 'E',
-    template: require('../templates/annotation_share_dialog.html'),
-    // @ngInject
-    controller: function ($scope, $element) {
-      var shareLinkInput = $element.find('input')[0];
+module.exports = {
+  controllerAs: 'vm',
```

Comment:
In follow up work I think we should get rid of `controllerAs` everywhere and just use the default `$ctrl`.